.factory/INSTALL.md

```
@@ -0,0 +1,50 @@
+1. **Add the superpowers marketplace:**
+   ```bash
+   droid plugin marketplace add https://github.com/obra/superpowers
+   ```
+
+2. **Install the plugin:**
+   ```bash
+   droid plugin install superpowers@superpowers
+   ```
+
+   Or use the interactive UI:
+   ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)